src/main/kotlin/solve/importer/view/DirectoryPathView.kt

```diff
@@ -0,0 +1,95 @@
+package solve.importer.view
+
+import javafx.beans.property.SimpleStringProperty
+import javafx.geometry.Insets
+import javafx.scene.control.OverrunStyle
+import javafx.scene.image.ImageView
+import javafx.stage.DirectoryChooser
+import solve.importer.ProjectParser.parseDirectory
+import solve.importer.controller.ImporterController
+import solve.importer.model.ImporterProject
+import solve.utils.loadImage
+import tornadofx.*
+import java.io.File
+
+class DirectoryPathView : View() {
+    private val controller: ImporterController by inject()
+    val path = SimpleStringProperty()
```

Comment:
Is it really required? May be just handle the original property of the controller changes?